tests/integration/conftest.py

```diff
@@ -160,7 +171,17 @@ def test_pod():
 
     # Wait until the pod is in ready state
     wait_result = subprocess.run(
-        ["kubectl", "wait", "--for", "condition=Ready", f"pod/{pod_name}", "--timeout", "60s"]
+        [
+            "kubectl",
+            "wait",
+            "--for",
+            "condition=Ready",
+            f"pod/{pod_name}",
+            "-n",
+            namespace,
+            "--timeout",
+            "60s",
```

Comment:
Can you increase timeout to avoid failed test in case of network issue? I will suggest to put 300 sec

Reply:
The CI pre-downloads the image and caches it before even running the integration tests. Also, since this code exists from long ago and isn't something added now, I'll leave it as such for now. During improving flaky tests for Spark K8s bundle, I can come back to it if it is really causing issues.